src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1323,25 +1324,30 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
         if (BlockRange().TryGetUse(node, &use))
```

Comment:
What happens if node is unused? Does LSRA remove it if it is side effect free?

Are there any cases it won't be removed and where codegen would fail because the relevant mask register isn't available?

Reply:
I haven't seen cases where that happens so far.  LSRA shouldn't remove them. I will add a note to check as a follow-up work.